addons/web/static/tests/views/kanban_tests.js

```diff
@@ -4483,6 +4483,48 @@ QUnit.module('Views', {
         kanban.destroy();
     });
 
+    QUnit.test('empty grouped kanban with sample data: cannot fold a column', async function (assert) {
```

Comment:
The test does not relate directly to your fix right?

Reply:
Non pas directement, c'était un fix scss a l origine, qui a un peu évolué (pour garder le dropdown ouvert) suite a la review de luc. J'ai voulu tester cette partie spécifiquement, sauf qu en fait on sait pas, car programmatiquement si je trigger un click sur 'Fold', ben ca fold (on bypass la regle pointer-events none si on fait comme ca). Du coup j ai juste gardé la premiere partie de mon test, mais qui passait deja avant (mais on ne le teste pas ailleurs).